lib/post_revisor.rb

```diff
@@ -601,7 +601,11 @@ def advance_draft_sequence
   def post_process_post
     @post.invalidate_oneboxes = true
     @post.trigger_post_process
-    DiscourseEvent.trigger(:post_edited, @post, self.topic_changed?, self)
```

Comment:
We can't change the arguments for triggers like this without having to go through a depreciation cycle since plugins might be using these triggers.

Reply:
How long does a "depreciation cycle" take?

Reply:
I didn't add depreciation. I reincluded that argument for now.